scripts/build-dist.sh

```
@@ -0,0 +1,553 @@
+
+# Strip YAML frontmatter from Markdown, return body only
+strip_frontmatter() {
+    local file="$1"
+    awk 'BEGIN{fm=0} /^---$/{fm++; next} fm>=2{print}' "$file"
+}
+
+# Convert a PitchDocs command .md to Gemini CLI .toml
+command_to_toml() {
+    local src="$1" dest="$2"
+    local desc body name
+    desc=$(frontmatter_val "$src" "description" | sed 's/: \$ARGUMENTS$//')
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)